packages/twenty-server/src/engine/core-modules/twenty-config/config-variables.ts

```diff
@@ -358,6 +358,15 @@ export class ConfigVariables {
   })
   EMAIL_SMTP_PASSWORD: string;
 
+  @ConfigVariablesMetadata({
+    group: ConfigVariablesGroup.Other,
+    description:
+      'When enabled, only workspace admins can create new workspaces. Ignored during initial setup when no workspace exists.',
+    type: ConfigVariableType.BOOLEAN,
+  })
+  @IsOptional()
+  IS_WORKSPACE_CREATION_LIMITED_TO_WORKSPACE_ADMINS = false;
```

Comment:
I was wrong in naming it this way, it's not Workspace Admins but server Admin, sorry